lib/killrvideo-docker-common/docker-compose.yaml

```diff
@@ -48,7 +49,7 @@ services:
   # Container to load KillrVideo schema and search config into DSE
   # Provides options to configure secure users as well
   dse-config:
-    image: killrvideo/killrvideo-dse-config:1.2.1
+    image: hadesarchitect/killrvideo-dse-config:disable-loopback
```

Comment:
Did you build a local version of killrvideo-dse-config to support removing loopback?  If so, we should get that one PR'd up into master and create a new version in Docker Hub we can then reference here.  Most likely version 1.2.2 or something after 1.2.1.